src/test/java/jenkins/metrics/RoundTripJCascMetricsTest.java

```diff
@@ -2,24 +2,26 @@
 
 import hudson.ExtensionList;
 import hudson.util.Secret;
-import io.jenkins.plugins.casc.misc.RoundTripAbstractTest;
+import io.jenkins.plugins.casc.misc.junit.jupiter.AbstractRoundTripTest;
 import jenkins.metrics.api.MetricsAccessKey;
-import org.jvnet.hudson.test.RestartableJenkinsRule;
+import org.jvnet.hudson.test.JenkinsRule;
+import org.jvnet.hudson.test.junit.jupiter.WithJenkins;
 
 import java.util.List;
 
 import static org.hamcrest.MatcherAssert.assertThat;
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertFalse;
-import static org.junit.Assert.assertTrue;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
 
 /**
  * Configuration, validation, export and configuration during restart of the configuration of the metrics plugin.
  */
-public class RoundTripJCascMetricsTest extends RoundTripAbstractTest {
+@WithJenkins
+class RoundTripJCascMetricsTest extends AbstractRoundTripTest {
 
     @Override
-    public void assertConfiguredAsExpected(RestartableJenkinsRule j, String configContent) {
+    protected void assertConfiguredAsExpected(JenkinsRule j, String configContent) {
```

Comment:
I guess the usage of the restartable jenkins rule was justified, why going back to the simple jenkins rule?